config.c

```diff
@@ -68,6 +68,7 @@ void finish_config(struct mako_config *config) {
 }
 
 void init_default_style(struct mako_style *style) {
+	style->min_width = 300;
```

Comment:
Sounds like this default would break existing config files setting `width` to something other than 300.

Reply:
Thanks for noticing, I'll add the ability to disable the feature if the minimum width is 0.